pegen/metagrammar.gram

```diff
@@ -54,4 +54,17 @@ atom:
     | NAME {NameLeaf(name.string) }
     | STRING {StringLeaf(string.string)}
 
-action: action=CURLY_STUFF {action}
+# Mini-grammar for the actions
+
+action: "{" stuffs "}" { stuffs }
+
+stuffs:
```

Comment:
Can you think of better names than `stuffs` and `stuff`? I meant them to be like `foo` and `bar` but never got around to inventing better names. Now's the time, I guess. :-)

Reply:
I updated the PR to use the uninspired `action_items`  and `action_item`. I was thinking about something with `atom`/`molecule` but I was unsure if it fits the convention with those. 

Please, advice if you find better names :)